src/handlers/assign/auto.ts

```diff
@@ -0,0 +1,73 @@
+import { Context } from "probot";
+import { getBotContext, getLogger } from "../../bindings";
+import { addAssignees } from "../../helpers";
+import { gitLinkedIssueParser } from "../../helpers/parser";
+import { Payload } from "../../types";
+
+// Use `context.octokit.rest` to get the pull requests for the repository
+export const getPullRequests = async (context: Context) => {
+  const logger = getLogger();
+  const payload = context.payload as Payload;
+  try {
+    const { data: pulls } = await context.octokit.rest.pulls.list({
```

Comment:
The functions/files to use `context.octokit.rest` needs to be placed in [`src/helpers`](https://github.com/ubiquity/bounty-bot/tree/development/src/helpers). don't want to be placed out of  the`helpers` directory.